src/components/Table/TableBody/TableBody.jsx

```diff
@@ -55,15 +55,18 @@ const propTypes = {
     onRowClicked: PropTypes.func,
     onApplyRowActions: PropTypes.func,
     onRowExpanded: PropTypes.func,
+    onApplyEdit: PropTypes.func,
   }).isRequired,
   /** What column ordering is currently applied to the table */
   ordering: PropTypes.arrayOf(
     PropTypes.shape({
       columnId: PropTypes.string.isRequired,
       /* Visibility of column in table, defaults to false */
       isHidden: PropTypes.bool,
+      editDataFunction: PropTypes.bool,
```

Comment:
why do we need editDataFunction boolena in the ordering section?

Reply:
My fault, in the beginning, I thought there was a connection between `hasRowSelection` and `hasEdit`, i.e. when table does the row-selection action and hide some columns, edit action on the above columns should be hiden too. But now, I remove editDataFunction from `ordering`, and find it is OK.